Mathlib/Mathport/Syntax.lean

```diff
@@ -180,9 +174,6 @@ syntax (name := iterate) "iterate" (num)? ppSpace tacticSeq : tactic
 -- Moved to Mathlib/Tactic/Basic.lean
 -- syntax (name := repeat') "repeat' " tacticSeq : tactic
 syntax (name := abstract) "abstract" (ppSpace ident)? ppSpace tacticSeq : tactic
-syntax (name := have'') "have " Term.haveIdLhs : tactic
```

Comment:
There is already a have tactic with more general syntax.  We also have let and suffices tactics in core.

Reply:
Nevermind, I've figure out what they were for.  I've added a comment.